demos/benchmark/lv_demo_benchmark.c

```diff
@@ -490,9 +584,64 @@ static uint32_t rnd_act;
  *   GLOBAL FUNCTIONS
  **********************/
 
+static void benchmark_context_init(benchmark_context_t * context)
```

Comment:
It is recommended to put the memory allocation and initialization of the context into one function to keep the dynamically allocated memory consistent with the life cycle of the object itself:
```
static benchmark_context_t * benchmark_context_create(void)
```